_posts/2022-09-30-release-table-store-0.2.1.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Apache Flink Table Store 0.2.1 Release Announcement"
+date: 2022-09-30T22:00:00.000Z
+categories: news
+authors:
+- JingsongLi:
+  name: "Jingsong Lee"
+
+excerpt: The Apache Flink Community is pleased to announce a bug fix release for Flink Table Store 0.2.
+
+---
+
+The Apache Flink Community is pleased to announce the first bug fix release of the Flink Table Store 0.2 series.
+
+This release includes 14 bug fixes, vulnerability fixes, and minor improvements for Flink Table Store 0.2.
```

Comment:
Please check and confirm whether [FLINK-29554](https://issues.apache.org/jira/browse/FLINK-29554) and [FLINK-29345](https://issues.apache.org/jira/browse/FLINK-29345) should be included and update the number accordingly if not.